R/deltaccd_plot.R

```diff
@@ -31,8 +33,9 @@ calcColorLimits = function(
 
 
 plotHeatmapSimple = function(ggObj, cLims) {
+  gene1 = gene2 = rho = NULL
```

Comment:
using `.data` means NULL assignment is unnecessary

Reply:
Fixed!